lms/envs/common.py

```diff
@@ -3552,3 +3552,6 @@ def _make_locale_paths(settings):
 # CDN LINK
 
 CDN_LINK = "https://static.philanthropyu.org/"
+
+# Google reCaptcha verify URL
+CAPTCHA_VERIFY_URL = "https://www.google.com/recaptcha/api/siteverify"
```

Comment:
Let's please move this in student_accounts/constants, because for now, I can't foresee its being used anywhere else. So instead of making core change, we should move it to our app.